openhtf/core/test_record.py

```diff
@@ -125,7 +125,7 @@ def _get_source_safely(obj):
     return ''
 
 
-class CodeInfo(mutablerecords.Record(
+class CodeInfo(mutablerecords.HashableRecord(
```

Comment:
not sure why this was changed?

Reply:
Changed to use Record.

Reply:
If I don't use HashableRecord on PY3 I see:

cls = <Record: PhaseDescriptor>

    def __hash__(cls):
        return hash(
            (cls.required_attributes,
>            frozenset(cls.optional_attributes.items())))
E       TypeError: unhashable type: 'CodeInfo'